src/components/screens/question-form.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardActionArea,
+  CardActions,
+  CardContent,
+  Checkbox,
+  TextField,
+  Typography
+} from "@mui/material";
+import type React from "react";
+import { useState } from "react";
+
+/**
+ * NewQuestionCard component
+ * Interface for the NewQuestionCard component
+ */
+
+interface NewQuestionProps {
+  handleAddQuestionSubmit: (
+    questionText: string,
+    options: { label: string; value: boolean }[]
+  ) => void;
+}
+
+const NewQuestionCard: React.FC<NewQuestionProps> = ({ handleAddQuestionSubmit }) => {
```

Comment:
No need for the React.FC typing here, use the Props type to type the destructured props e.g. 
({ handleAddQuestionSubmit }: Props)- in this case you can rename NewQuestionProps to Props. E.g. see how this is done in TimebankPieChart component.